drape_frontend/drape_measurer.cpp

```diff
@@ -14,17 +14,15 @@ void DrapeMeasurer::StartBenchmark()
   using namespace std::chrono;
   m_isEnabled = true;
 
-  auto currentTime = std::chrono::steady_clock::now();
-
 #ifdef GENERATING_STATISTIC
-  m_startScenePreparingTime = currentTime;
+  m_startScenePreparingTime = steady_clock::now();
```

Comment:
Please, keep an old behavior, i.e. create a named variable and set it to steady_clock::now.